booking/src/test/java/org/dcsa/conformance/standards/booking/party/BookingStateTest.java

```diff
@@ -0,0 +1,28 @@
+package org.dcsa.conformance.standards.booking.party;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class BookingStateTest {
+
+  @Test
+  void testFromString_validInput() {
+    assertEquals(BookingState.START, BookingState.fromString("START"));
+    assertEquals(BookingState.RECEIVED, BookingState.fromString("RECEIVED"));
+  }
+
+  @Test
+  void testFromString_invalidInput() {
+    assertThrowsExactly(
+        IllegalArgumentException.class, () -> BookingState.fromString("INVALID_STATE"));
+    assertThrowsExactly(IllegalArgumentException.class, () -> BookingState.fromString(null));
+    assertThrowsExactly(IllegalArgumentException.class, () -> BookingState.fromString(" "));
+    assertThrowsExactly(IllegalArgumentException.class, () -> BookingState.fromString(""));
+  }
+
+  @Test
+  void testFromString_caseInsensitiveInput() {
+    assertThrowsExactly(IllegalArgumentException.class, () -> BookingState.fromString("start"));
+    assertThrowsExactly(IllegalArgumentException.class, () -> BookingState.fromString("received"));
```

Comment:
Please use `assertThrows` if that also fixes the job. 
In `BookingCancellationStateTest.java` you did that as well, which is 👍🏻